cfp.html

```diff
@@ -132,6 +132,20 @@ <h2>Topics of Interest</h2>
 
                         </br>
 
+                        <h2>Open Science Policy</h2>
+                        <div class="description-one">
+                            <p align="justify">
+                                Openness in science is key to fostering progress via transparency, reproducibility and replicability.
```

Comment:
is "a" key "for"?
", and"

Reply:
> is "a" key "for"?

I don't think so, because I think that openness is the only key for those things.

>  ", and"

I will change that.

Reply:
Maybe "the key" then? I think it's weird to have "key" without it..

Reply:
I think you are right. I have made the change.